protocol/0056-REWA-rewards_overview.md

```diff
@@ -466,30 +466,143 @@ Market has been trading but not yet eligible for proposer bonus.
+* The value of `marketCreationQuantumMultiple` is `10^6` and `quantum` for both `USDT` and `DAI` is `1`. 
+* Setup and fund recurring reward account transfers using the market_proposer metric and `USDT` metric asset:
+    * Transfer 10000 $VEGA to `ETHUSDT | market creation | $VEGA` 
+* start trading in the market such that traded value for fee purposes in USDT is less than 10^6
+* During epoch 2 let the traded value on ETHUSDT and BTCUSDT be greater than 10^6
+* During epoch 3, setup and fund recurring reward account transfers using the market_proposer metric and `USDT` metric asset:
+    * Transfer 10000 $VEGA to `all | market creation | $VEGA` 
+
+#### Expectation
+At the end of epoch 2, 10000 VEGA rewards should be distributed to only the ETHUSDT creator.
+    The general account balance of the ETHUSDT creator should be 10000.
+    The general account balance of the BTCDAI creator should be 0.
+    The reward pool balance should be 0.
+
+At the end of epoch 3, 10000 VEGA should be distributed split between the BTCDAI creator and the ETHUSDT creator.
+    The general account balance of the ETHUSDT creator should be 15000.
+    The general account balance of the BTCDAI creator should be 5000.
+    The reward pool balance should be 0.
+
+
 
 
 
+### Distributing market creation rewards - Multiple markets eligible, one already paid, specified asset (<a name="0056-REWA-047" href="#0056-REWA-047">0056-REWA-047</a>)
+#### Rationale 
+A market reward pool funded with the same asset by the same party with different market scopes should pay to all markets even if already paid
+#### Setup
+* Setup a market ETHUSDT settling in USDT.
+* Setup a market BTCUSDT settling in USDT with a different party.
+* The value of `marketCreationQuantumMultiple` is `10^6` and `quantum` for `USDT`is `1`. 
+* Setup and fund recurring reward account transfers for the market ETHUSDT, specifying `USDT` markets for the metric asset:
+    * Transfer 10000 $VEGA to `ETHUSDT | market creation | $VEGA` 
+* start trading in the market such that traded value for fee purposes in USDT is less than 10^6
+* During epoch 2 let the traded value on ETHUSDT and BTCUSDT be greater than 10^6
+* During epoch 3, setup and fund recurring reward account for all markets, leaving a blank metric asset, with the same party:
+    * Transfer 10000 $VEGA to `all | market creation | $VEGA` 
+
+#### Expectation
+At the end of epoch 2 the full 10000 VEGA rewards should be distributed to only the ETHUSDT creator. At the end of epoch 3 the full 10000 VEGA rewards should be distributed to the BTCUSDT creator.
 
+At the end of epoch 2, 10000 VEGA rewards should be distributed to only the ETHUSDT creator.
+    The general account balance of the ETHUSDT creator should be 10000.
+    The general account balance of the BTCUSDT creator should be 0.
+    The reward pool balance should be 0.
 
+At the end of epoch 3, 10000 VEGA should be distributed split between the BTCUSDT creator and the ETHUSDT creator.
```

Comment:
I think the balances below are incorrect. I'd expect the balance of the proposer of ETHUSDT to be 15k and of the proposer of BTCUSDT to be 5k. also, when specifying the markets it's worth mentioning that you want them to be proposed by different parties or the same.

Reply:
Thanks for catching, I even said as much in the line above so not sure where the 10000 each came from! Updating